.storybook/main.js

```diff
@@ -1,26 +1,50 @@
 module.exports = {
-	stories: [
-		'../src/**/*.stories.mdx',
-		'../src/**/*.stories.@(js|jsx|ts|tsx)'
-	],
+	stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
 	addons: [
 		{
 			name: '@storybook/preset-scss',
 			options: {
 				sassLoaderOptions: {
 					sassOptions: {
-						includePaths: ["src"]
+						includePaths: ['src'],
 					},
-				}
-			}
+				},
+			},
 		},
 		'@storybook/addon-links',
-		'@storybook/addon-essentials'
+		'@storybook/addon-essentials',
 	],
 	framework: '@storybook/react',
 	core: {
-		builder: 'webpack5'
+		builder: 'webpack5',
 	},
 	staticDirs: ['../public'],
 	typescript: { reactDocgen: false },
+	// Extend Storybook's internal webpack compiler to also compile components imported from the avo2-components repo
+	webpackFinal: (config) => {
```

Comment:
Dit ga je niet meer nodig hebben eens https://github.com/viaacode/hetarchief-client/pull/12 er door is.
Hierdoor hoef je niet meer te importen vanuit src uit sommige packages.